tools/nni_cmd/launcher.py

```diff
@@ -271,6 +271,8 @@ def set_experiment(experiment_config, mode, port, config_file_name):
         request_data['tuner'] = experiment_config['tuner']
         if 'assessor' in experiment_config:
             request_data['assessor'] = experiment_config['assessor']
+    if experiment_config.get('versionCheck') is not None:
```

Comment:
if experiment_config.get('versionCheck'):

Reply:
experiment_config['versionCheck'] could be true, false or None, I need to set request_data values if it is not None. If change to `if experiment_config.get('versionCheck'):`, it will not set values when experiment_config['versionCheck'] is false.

Reply:
got it.